sources/web/datalab/server.ts

```diff
@@ -156,6 +150,12 @@ function handleRequest(request: http.ServerRequest,
     return;
   }
 
+  // /setting updates a per-user setting.
+  if (path.indexOf('/setting') == 0) {
```

Comment:
Can we use "/_setting" instead? "/setting" seems a bit too common. The chance of overlapping with jupyter requests is lower with "/_setting".